frontend/src/js/controllers/updateProfileCtrl.js

```diff
@@ -33,6 +33,28 @@
             $rootScope.loaderTitle = '';
             vm.updateprofileContainer.removeClass('low-screen');
         };
+
+        // To get the previous profile data
+        var parameters = {};
+        parameters.url = 'auth/user/';
+        parameters.method = 'GET';
+        parameters.token = userKey;
+        parameters.callback = {
+            onSuccess: function(response) {
+                var status = response.status;
+                var result = response.data;
+                if (status == 200) {
+                    vm.user = result;
+                }
+
+            },
+            onError: function(response) {
+                $rootScope.notify("error", "Error in loading old profile, please try again.");
```

Comment:
@aka-jain I had made the chenges. Please refer to this [link](https://github.com/Cloud-CV/EvalAI/pull/801/files#diff-6337108b641fb2b83e8b675eaf1572d0R50). :-)